src/Pelagos/Bundle/AppBundle/Resources/views/DataDiscovery/template-dataset-row.html.twig

```diff
@@ -0,0 +1,25 @@
+<table hidden>
```

Comment:
the hidden tag is new in HTML 5. Using this tag may not give the desired result. When cloned, this tag will be copied too. You'd have to remove it, but the same would go when using a class.